cluster-scope/base/clusterloggings/clusterLogging.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
```

Comment:
Please move the namespace here into the `kustomization.yaml` so we are consistent across all the manifests.

Reply:
fixed, done.